daffodil-core/src/main/scala/org/apache/daffodil/dsom/Term.scala

```diff
@@ -732,9 +728,8 @@ trait Term
   final def possibleFirstChildElementsInInfoset: Seq[ElementBase] = LV('possibleFirstChildElementsInInfoset) {
     val pfct = possibleFirstChildTerms
     val firstChildren = pfct.flatMap {
-      case e: ElementBase if e.isHidden => Nil
       case e: ElementBase => Seq(e)
-      case s: SequenceTermBase if s.isHidden => Nil
+      case gr: GroupRef if gr.asModelGroup.isHidden => Nil
```

Comment:
Just curious how difficult it would be to move isHidden (or isHiddenGroupRef, agreed that's amore clear name) to GroupRef? Now that you mention it, that does feel like that's maybe a more natural place for this property.

Reply:
I decided to move isHidden to GroupRef, as isHidden inherited from MG was only ever possibly true for these cases.